doc/cli.markdown

```diff
@@ -975,6 +975,7 @@ Examples:
 	$ balena ssh 7cf02a6 --port 8080
 	$ balena ssh 7cf02a6 -v
 	$ balena ssh 7cf02a6 -s
+	$ balena ssh 7cf02a6 -s -c 'date'
```

Comment:
Can we get away with not needing the `-c`?

Reply:
@CameronDiver don't think we can do multiple `COMMAND` in Capitano? And the UUID is already the command? I'd love to get rid of that too :/ Will try some more

Reply:
iirc, Capitano had the "concepts" of parameters and options, and there could be multiple parameters, though I'm not 100% sure. So in `balena ssh 7cf02a6 'date' -s`, there's be two parameters (`7cf02a6` and `'date'`) and one option (`-s`). I could well be wrong, though!

Reply:
@pdcastro @CameronDiver updated this, it's now an optional command